5.6/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:3.4
+
+RUN apk update \
+    && apk add --no-cache \
+    php \
+    php5-apcu \
+    php5-bcmath \
+    php5-ctype \
+    php5-curl \
+    php5-dom \
+    php5-exif \
+    php5-gd \
+    php5-iconv \
+    php5-intl \
+    php5-json \
+#    php5-libsodium \

```

Comment:
php5-libsodium this package exist you can uncomment: https://pkgs.alpinelinux.org/package/edge/testing/x86/php5-libsodium